Testshell/main.cpp

```diff
@@ -146,6 +132,68 @@ TEST_F(TestShellFixture, ReadLbaOverThanMaxOfInt) {
   ts.executeCommand(command);
 }
 
+TEST_F(TestShellFixture, TestScript1FAIL) {
+  Command cmd{"1_FullWriteAndReadCompare", {"0xAAAABBBB"}};
```

Comment:
"1_FullWriteAndReadCompare"나 "1_" 같은 값들은 ```TestShellFixture``` class 안에서 ```TEST_SCRIPT_1_FULLNAME```, ```TEST_SCRIPT_1_SHORTCUT```와 같은 상수로 선언해주시는게 보기 더욱 깔끔할 것 같습니다.
```suggestion
  Command cmd{TEST_SCRIPT_1_FULLNAME, {"0xAAAABBBB"}};
```

Reply:
좋은 방법인 것 같습니다. 감사합니다